index.js

```diff
@@ -43,5 +43,32 @@ module.exports = {
     'string-no-newline': true,
     'unit-no-unknown': true,
     'value-no-vendor-prefix': true,
+    'scale-unlimited/declaration-strict-value': [
+      [
+        // Colors should always be defined from variables or functions.
+        '/color/',
+        'fill',
+        'stroke',
+        // Font tokens should come from our design tokens.
+        'font-family',
+        'font-size',
+        'font-weight',
+        // Spacing should use a consistent scale rather than hard-coded values.
+        '/margin/',
+        '/padding/',
+        'gap',
+        // Consistently using variables for z-index allows us to define the order of the values globally.
+        'z-index',
+      ],
+      {
+        ignoreKeywords: [
```

Comment:
According to the stylelint-declaration-strict-value README, `ignoreKeywords` has been deprecated in favor of `ignoreValues`.

```suggestion
        ignoreValues: [
```